hypothesis-python/src/hypothesis/extra/ghostwriter.py

```diff
@@ -877,6 +877,13 @@ def maker(sub_property: str, args: str, body: str, right: str = None) -> None:
                 identity = find(operands, lambda x: True)
             except Exception:
                 identity = "identity element here"  # type: ignore
+        # If the repr of this element is invalid Python, stringify it - this
+        # can't be executed as-is, but at least makes it clear what should
+        # happpen.  E.g. type(None) -> <class 'NoneType'> -> quoted.
+        try:
+            compile(repr(identity), "<string>", "exec")
```

Comment:
This doesn't actually execute, correct? It simply tries to create a code object that _could_ be executed by `exec`? Just checking, because actually executing would be scary.

Reply:
That's correct, and I'll add a comment to this effect.